content/reports/240209-contributors.md

```diff
@@ -0,0 +1,28 @@
+---
+type: contributors
+date: 2024-02-09
+description: |
+  Weekly meeting of the TDP contributors to discuss open PRs and particularly of the TDP 1.1 release.
+---
+
+# TDP contributors meeting notes
+
+Weekly meeting of the TDP contributors to discuss open PRs and particularly of the TDP 1.1 release.
```

Comment:
Remove that line which doesnt bring values.

Start with a summary of the major activities of the meeting (eg 1 to 3 main topics discussed)